DirectXTex/BC.cpp

```diff
@@ -9,7 +9,7 @@
 // http://go.microsoft.com/fwlink/?LinkId=248926
 //-------------------------------------------------------------------------------------
 
-#include "directxtexp.h"
```

Comment:
For testing I put the ``#pragma clang diagnostic ignore`` suppressions into the pch here, but those suppressions weren't in place before it was included, so you got a bunch of ``-Wnonportable-include-path`` warnings, so I changed these to make it less noisy...